src/libcore/time.rs

```diff
@@ -45,7 +58,7 @@ const MICROS_PER_SEC: u64 = 1_000_000;
 ///
 /// let ten_millis = Duration::from_millis(10);
 /// ```
-#[stable(feature = "duration", since = "1.3.0")]
+#![stable(feature = "duration_core", since = "1.24.0")]
```

Comment:
Why add the `!` here...

```
[00:03:27] error: an inner attribute is not permitted following an outer doc comment
[00:03:27]   --> /checkout/src/libcore/time.rs:61:3
[00:03:27]    |
[00:03:27] 61 | #![stable(feature = "duration_core", since = "1.24.0")]
[00:03:27]    |   ^
[00:03:27]    |
[00:03:27]    = note: inner attributes, like `#![no_std]`, annotate the item enclosing them, and are usually found at the beginning of source files. Outer attributes, like `#[test]`, annotate the item following them.
```

Reply:
…I copy-pasted and shouldn't have.